tracer/src/Datadog.Trace/LibDatadog/NativeInterop.cs

```diff
@@ -0,0 +1,67 @@
+// <copyright file="NativeInterop.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Datadog.Trace.LibDatadog;
+
+internal class NativeInterop
+{
+    private const string DllName = "LibDatadog";
+
+    internal static class Exporter
+    {
+        // [DllImport(DllName, EntryPoint = "ddog_trace_exporter_new")]
+        // internal static extern ErrorHandle Create(out IntPtr outHandle, SafeHandle config);
+
+        [DllImport(DllName, EntryPoint = "ddog_trace_exporter_error_free")]
+        internal static extern void ReleaseError(IntPtr error);
+
+        [DllImport(DllName, EntryPoint = "ddog_trace_exporter_free")]
+        internal static extern void Release(IntPtr handle);
```

Comment:
we can probably add a dummy test in Datadog.Trace.Tests, which calls with some ptr. It will fail but we can make sure library is loaded on all platforms and right entry point is invoked.

Reply:
This was to see if the native part was able to rewrite it.
The tests should be added when you add data-pipeline and/or config support

Reply:
After rewrite, were you able to call any of the methods?

I want to this on my end as well - just to see library integration is perfect and we can focus on  calling the methods.